opengever/maintenance/scripts/set_registry_value.py

```diff
@@ -0,0 +1,67 @@
+"""
+Allows to set an arbitrary registry record to a particular value.
+
+The <value> is supposed to be specified as a Python literal.
+
+Example Usage:
+
+    bin/instance run set_registry_value.py <record_dotted_name> <value>
+    bin/instance run set_registry_value.py opengever.base.interfaces.IGeverUI.is_feature_enabled True
+"""
+from ast import literal_eval
+from opengever.maintenance.debughelpers import setup_app
+from opengever.maintenance.debughelpers import setup_plone
+from plone.registry.interfaces import IRegistry
+from zope.component import getUtility
+import argparse
+import sys
+import transaction
+
+
+def cast_value(value, pytype):
+    value = literal_eval(value)
+    if isinstance(value, pytype) or value is None:
+        return value
+
```

Comment:
didn't you want to try to cast here: `pytype(value)`?

Reply:
Something along those lines is what I first went with, but:
- `bool(4)` would happily cast to `True`
- plone.registry happily accepts a value of `4` for a `bool` record too
- As would `bool("False")`
- Casting wouldn't work at all for something like `[1, 2, 3]`, but `literal_eval` can handle that just fine

So, given all these, I went with an approach that first parses the Python literal (or rejects it, if not a valid literal), and then asserts that it is exactly of the expected type (which it should be without any further casting).

This makes it more strict, and avoids ambigous cases like `int(True)` or `bool(1)` being casted implicitly, which probably mean someone is using it wrong.